src/App.tsx

```diff
@@ -1,26 +1,448 @@
 /* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import cn from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
+import { UserWarning } from './UserWarning';
+import {
+  addTodos,
+  deleteTodos,
+  getTodos,
+  updateTodos,
+  USER_ID,
+} from './api/todos';
+import { Todo } from './types/Todos';
+import { Filter } from './types/FilterEnum';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filterStatus, setFilterStatus] = useState<Filter>(Filter.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [deletingTodosIds, setDeletingTodosIds] = useState<number[]>([]);
```

Comment:
looks like you can just create one state for todosIds instead of `deletingTodosIds` and `updatingTodosIds`